auslib/db.py

```diff
@@ -871,7 +887,7 @@ def _splitColumns(self, columns):
         base_columns = {}
         condition_columns = {}
         for k in columns:
-            if k in itertools.chain(*self.conditions.condition_groups):
+            if k in itertools.chain(*self.conditions.condition_groups.values()):
```

Comment:
Nit: `k` seems a bit odd if we take the values of a condition group. How about something like `condition_type`?